### ISSUE TITLE
Allow control of the default environment name

### ISSUE THREAD

**Issue by stuartleeks** [Lambda3/vsts-build-task-docker#1]
As per the marketplace instructions, 

> ```
> The environment will affect the Dockerfile that is used during build. If you select Debug, the build task will search for Dockerfile, otherwise it will search for Dockerfile.<environment>.
> ```

It would be really helpful to be able to specify which environment is default. E.g. if I set release to the default environment then when the environment is set to release it would use Dockerfile, and for any other environment it would use Dockerfile.<environment>

The default value for the default environment could be "debug" to preserve existing behaviour unless someone explicitly changes it

**Comment by giggio** [comment]
@stuartleeks Right now `Debug` is the default. What would you like to see changed?

**Comment by stuartleeks** [comment]
I'd like to be able to customise the default value :-)

E.g. I'd like to be able to have the `Release` environment use `Dockerfile`, and the `Debug` environment user `Dockerfile.debug`

Does that make sense?

**Comment by giggio** [comment]
Yes, it does. The reason I did not do that, is that the syntax highlighting (and other tools support) usually stops working with the `.debug` or `.release` extension. I saw you send a PR, I will check it out now.

**Comment by stuartleeks** [comment]
I use Visual Studio Code for a lot of Dockerfile editing and it is well-behaved with `Dockerfile.something` filenames, so I just assumed others were too :-)
